src/js/components/Autocomplete.js

```diff
@@ -157,6 +166,7 @@ class Autocomplete extends Component {
         isShowFullResults = false,
         value,
         searchOptions,
+        isHome,
```

Comment:
Is it me only who can see `isHome` is also part of state object or its just the rearranging of object keys?